src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -129,9 +129,19 @@ protected function describeContainerDefinition(Definition $definition, array $op
     /**
      * {@inheritdoc}
      */
-    protected function describeContainerAlias(Alias $alias, array $options = array())
+    protected function describeContainerAlias(Alias $alias, array $options = array(), ContainerBuilder $builder = null)
     {
-        $this->writeData($this->getContainerAliasData($alias), $options);
+        if (!isset($options['id']) || !$builder) {
+            return $this->writeData($this->getContainerAliasData($alias), $options);
+        }
+
+        $serviceId = $options['id'];
+
+        do {
+            $serviceId = (string) $builder->getAlias($serviceId);
```

Comment:
It would be indeed better but it doesn't look possible, AFAIK there's no way to handle chained aliases, it direct gives the original id.
Given `alias1` is an alias of `original` and, `alias2` is an alias of `alias1`, running `debug:container alias2` currently gives 

> This service is an alias for the service "original", 

and not

>This service is an alias for the service "alias1"

:/